apiserver/pkg/manager/resource_manager.go

```diff
@@ -20,9 +20,9 @@ const DefaultNamespace = "ray-system"
 // kubernetes objects and potential db objects underneath operations should be encapsulated at this layer
 type ResourceManagerInterface interface {
 	CreateCluster(ctx context.Context, apiCluster *api.Cluster) (*v1alpha1.RayCluster, error)
-	GetCluster(ctx context.Context, clusterName string) (*v1alpha1.RayCluster, error)
+	GetCluster(ctx context.Context, clusterName string, namespace string) (*v1alpha1.RayCluster, error)
 	ListClusters(ctx context.Context) ([]*v1alpha1.RayCluster, error)
```

Comment:
what's the plan for the list operation? I assume user may not have access to list cluster across namespaces? Should we limit to namespace scope?

Reply:
Yeah I agree. Having namespace scope for the list op makes more sense here. 

I can work on this after this PR is merged.

Reply:
sounds good. Please help create an issue to track it later.